stimela/commands/run.py

```diff
@@ -166,7 +166,7 @@ def run(what: str, parameters: List[str] = [], dry_run: bool = False, profile: b
                 try:
                     stimela.CONFIG.lib.recipes[name] = OmegaConf.merge(RecipeSchema, value)
                 except Exception as exc:
-                    log_exception(f"error in definition of recipe '{name}'", exc)
+                    log_exception(f"error in definition of recipe '{value}'", exc)
```

Comment:
@Mulan-94 won't this dump the entire recipe into the exception string? Doesn't that make it super verbose and unreadable?

Reply:
I'm not really sure. I changed it to the current state because I was getting an error message like:

`error in definition of recipe 'name'`

for an error in my recipe, which was not very clear. I assumed the idea was to pass whatever is held by the  'name' key. But, I haven't tested  how much can be dumped, shall I change it back?

Reply:
Well it should have been reporting the name of the top level recipe section, not just literally ``'name'``. Unless your top-level section was actually called "name"?